scripts/rollup/results.json

```diff
@@ -400,4 +400,4 @@
       "gzip": 525
     }
   ]
-}
\ No newline at end of file
+}
```

Comment:
I think this file should not be checked in. It should be gitignored.

Locally, instead of comparing to it during the run, we should locate a special cache folder (like `babel-loader` uses for caching). Inside that folder, we should have `{sha}.json` for every build done locally (that hasn't been purged from the cache yet).

When we build with a clean git state, we can copy the JSON as `{sha.json}` into the cache. On every build (clean or not) we should also look up `{merge-base-sha}.json` in local cache. If it exists, that's what we compare against. Otherwise, we don't print the stats locally.

Would that make sense?

Reply:
(It's not a blocker though if we can get the remote results to be right at least)

Reply:
I agree, it should be gitignored 👍 

I'm about to head into a meeting but I'll gonna take a look into why the results I'm seeing locally are different in general, maybe a node_modules introduced by Danger changes the results?